index.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
-var isFinite = require('is-finite');
+module.exports = (n, str) => {
+	str = str || ' ';
 
-module.exports = function (str, n) {
 	if (typeof str !== 'string') {
 		throw new TypeError('Expected `input` to be a string');
```

Comment:
Which one do you prefer?

1. Expected `input` to be a `string`, got `number`
2. Expected `input` to be a string, got `number`
3. Expected `input` to be a string, got number

Might prefer 1 the most